src/app/modules/main/profile_section/wallet/accounts/view.nim

```diff
@@ -28,6 +32,41 @@ QtObject:
     result.accountsVariant = newQVariant(result.accounts)
     result.keyPairModel = newKeyPairModel()
 
+  proc getSelectedKeyPair*(self: View): QVariant {.slot.} =
+    if self.selectedKeyPair.isNil:
+      return newQVariant()
+    return newQVariant(self.selectedKeyPair)
+
+  proc selectedKeyPairChanged(self: View) {.signal.}
+
+  QtProperty[QVariant] selectedKeyPair:
+    read = getSelectedKeyPair
+    notify = selectedKeyPairChanged
+
+  proc getSelectedAccount*(self: View): QVariant {.slot.} =
+    if self.selectedAccount.isNil:
+      return newQVariant()
+    return newQVariant(self.selectedAccount)
+
+  proc selectedAccountChanged(self: View) {.signal.}
+
+  QtProperty[QVariant] selectedAccount:
+    read = getSelectedAccount
+    notify = selectedAccountChanged
+
+  proc selectItem*(self: View, keyUid: string, address: string) {.slot.} =
```

Comment:
@Cuteivist could you change the name to `setSelectedAccount`?

Don't need to change this, but also its declaration should be `proc setSelectedAccount*(self: View, address: string) {.slot.} =` since the address is unique.